assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver-secret.yaml

```diff
@@ -14,4 +8,23 @@ data:
   etcd-client.key: "{{ .Values.apiserver.etcdClientKey }}"
   aggregation-ca.crt: "{{ .Values.apiserver.aggregationCaCert }}"
   aggregation-client.crt: "{{ .Values.apiserver.aggregationClientCert }}"
-  aggregation-client.key: "{{ .Values.apiserver.aggregationClientKey }}"
\ No newline at end of file
+  aggregation-client.key: "{{ .Values.apiserver.aggregationClientKey }}"
+{{- end -}}
+# Value of "token" is composed by injecting all values into kube-apiserver-secret.yaml template and
+# then calculating sha256 sum of it, so it will be different for each cluster and additionally will be
+# changed every time certificate or secret template changes.
+{{- define "token" -}}
+{{ include "secrets" . | sha256sum }}
+{{- end }}
+{{- define "token-auth-file" -}}
+{{ template "token" . }},nobody,nobody
+{{- end }}
```

Comment:
Do you happen to have a link handy for these helm things?

Reply:
https://helm.sh/docs/chart_template_guide/named_templates/

Reply:
Thanks!